src/transformers/pipelines/visual_question_answering.py

```diff
@@ -15,6 +15,11 @@
 logger = logging.get_logger(__name__)
 
 
+class ModelType(ExplicitEnum):
+    CLASSIFIER = "classifier"
+    GIT = "GIT"
```

Comment:
Maybe we can make a distinction between classifier vs generative. GIT is a generative model